x11-wm/mutter/mutter-9999.ebuild

```diff
@@ -64,7 +64,6 @@ RDEPEND="
 	>=gnome-base/gsettings-desktop-schemas-47_beta[introspection?]
 	>=dev-libs/glib-2.81.1:2
 	gnome-base/gnome-settings-daemon
-	>=dev-libs/json-glib-0.12.0[introspection?]
```

Comment:
fwiw, this got dropped upstream via https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/3354 for mutter-46.0; and the introspection USE dep was added by me in some ancient bump, seemingly wrongly or maybe it was getting exposed in public API and GIR file.